app/requirements.txt

```diff
@@ -1 +1,2 @@
-buildkite-sdk=0.2.0
+buildkite-sdk==0.2.0
```

Comment:
Should we remove this if it isn't needed?

Reply:
Replaced with a dependency that's actually used